Description: drop one unstable test
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1022421
Forwarded: not-needed
Last-Update: 2022-10-24

--- a/sinonjsfake-timers/test/fake-timers-test.js
+++ b/sinonjsfake-timers/test/fake-timers-test.js
@@ -3872,6 +3872,7 @@
                 this.clock.uninstall();
             });
 
+            /*
             it("should replace the getEntries, getEntriesByX methods with noops that return []", function () {
                 if (typeof Performance === "undefined") {
                     return this.skip();
@@ -3905,6 +3906,7 @@
                 delete Performance.prototype.getEntriesByName;
                 delete Performance.prototype.getEntriesByTime;
             });
+            */
         }
 
         /* eslint-disable mocha/no-setup-in-describe */
